CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Changelog
 
   **deprecated** `CacheManager::tagResponse` in favor of `TagHandler::addTags`
 * **2015-05-08** Added configuration option for custom proxy client (#208)
+* **2015-05-08** Added support for a simple Etag header in the header configuration (#207)
```

Comment:
hm, habit i guess. but as the date is when we started working on the feature rather than when it is merged, this is indeed rather pointless. i would not even add a date for the release i think - one can easily check git to know that, or see it on packagist.

i will remove the other dates in master once we merged the PRs, to avoid merge conflicts.